packages/nitro-protocol/package.json

```
@@ -46,7 +46,7 @@
     "docgen": "npx solidoc",
     "prettier:check": "npx prettier --check '{contracts,src,test}/**/*.{ts,tsx,sol}'",
     "prettier:write": "npx prettier --write '{contracts,src,test}/**/*.{ts,tsx,sol}'",
-    "startGanache": "node bin/start-ganache.js",
+    "startGanache": "npx tsc ganache/index.ts && NODE_ENV=dev npx node ganache/index.js",
```

```suggestion
    "startGanache": "NODE_ENV= npx ts-node ganache/index.js",
```

I think this should work (as long as `ts-node` is a devDep).

With this change, you can revert the change to `.gitignore`.

---

Can we not direct `ts-node` to a `.ts` file?

Seems weird that we'd need to check in a `.js` file if we're using `ts-node` to begin with.

---

that was a typo: should have been `ganache/index.ts`